c-deps/libroach/ccl/db_test.cc

```diff
@@ -82,65 +82,3 @@ TEST(LibroachCCL, DBOpen) {
     DBClose(db);
   }
 }
-
-TEST(LibroachCCL, EncryptedEnv) {
```

Comment:
This is moved without modification (except for the name) to `encrypted_env_test.cc`.